logstash-output-webhdfs.gemspec

```diff
@@ -0,0 +1,28 @@
+Gem::Specification.new do |s|
+
+  s.name            = 'logstash-output-webhdfs'
+  s.version         = '0.0.1'
```

Comment:
Done in https://github.com/dstore-dbap/logstash-output-webhdfs-1/commit/09b929cb82a8a4ecb6ced276589b7a3f2e2ed9f4